base/sparse/sparsematrix.jl

```diff
@@ -2458,11 +2461,14 @@ function setindex!{Tv,Ti,T<:Integer}(A::SparseMatrixCSC{Tv,Ti}, B::SparseMatrixC
     colptrB = B.colptr; rowvalB = B.rowval; nzvalB = B.nzval
```

Comment:
I think I messed up the rebase a bit. That was supposed to go in the (potential) later PR that removes manual hoistings. I will change back to using the `colptrB` variables.